src/violation.rs

```diff
@@ -66,8 +66,8 @@ impl ViolationWithContext {
 #[macro_export]
 macro_rules! violation {
     ($($variant:ident => $value:expr),* $(,)?) => {
-        proc_macros::builder_args!(
-            $crate::violation::ViolationBuilder,
+        $crate::builder_args!(
```

Comment:
Oh wait, you're already using `$crate` here, so I guess you knew about it. Is this is a different situation than above? Or maybe it's only for declarative macros.

Reply:
Ya see other reply, I don't think this is something I tried/tried googling or anything (it does seem like something that would "come up" when writing proc macros eg why `$crate` exists (at least for declarative macros))